decryptor/mysql/type_conversion.go

```diff
@@ -1,13 +1,46 @@
 package mysql
 
 import (
+	"github.com/sirupsen/logrus"
+
 	"github.com/cossacklabs/acra/decryptor/base"
 	"github.com/cossacklabs/acra/decryptor/base/type_awareness"
 	base_mysql "github.com/cossacklabs/acra/decryptor/mysql/base"
 	"github.com/cossacklabs/acra/encryptor/config"
 	"github.com/cossacklabs/acra/encryptor/config/common"
 )
 
+// TypeConfiguration represent configurations details of specific type
+type TypeConfiguration struct {
+	// https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_basic_character_set.html
+	Charset      uint16
+	ColumnLength uint32
+	// https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_com_query_response_text_resultset_column_definition.html
+	Decimal uint8
+}
+
+// TypeConfigurations contains specific info for used in TA types
+var TypeConfigurations = map[base_mysql.Type]TypeConfiguration{
+	base_mysql.TypeString: {
```

Comment:
let's copy here link with doc about charset and copy here query used to get charsets:
```
SELECT id, collation_name FROM information_schema.collations ORDER BY id;
```
for case when documentation will be unavailable but still be able to know how to get this charsets